simulate-travis.py

```diff
@@ -97,6 +97,12 @@
                 then we run independent mulled-build tests on
                 a successfully-built recipe. Use this argument to disable this
                 behavior''')
+ap.add_argument('--disable-local', action='store_true', help='''If you are on
```

Comment:
Should we rename these to `--disable-linux-tests` and `disable-osx-tests`? `local` and `docker` does not seem to be very meaningful.